server/app/models/AccountModel.java

```diff
@@ -177,14 +177,14 @@ public void removeAdministeredProgram(ProgramDefinition program) {
 
   /**
    * Returns the name, as a string, of the most-recently created Applicant associated with this
-   * Account. There is no particular reason for an Account to have more than one Applicant - this
-   * was a capability we built but did not use - so the ordering is somewhat arbitrary /
-   * unnecessary.
+   * Account. Or the email if no name is associated with the applicant. There is no particular
+   * reason for an Account to have more than one Applicant - this was a capability we built but did
+   * not use - so the ordering is somewhat arbitrary / unnecessary.
    */
   public String getApplicantName() {
```

Comment:
Hmm, was just taking a quick look before sending this out, and I wonder if I should rename this to getApplicantDisplayName() as well?